prog/vm/nexus.rb

```diff
@@ -542,8 +542,7 @@ def write_params_json
 
       # If there is a load balancer setup, we want to keep the network setup in
       # tact for a while
-      action = vm.load_balancer ? "delete_keep_net" : "delete"
-      host.sshable.cmd("sudo host/bin/setup-vm #{action} #{q_vm}")
+      host.sshable.cmd("sudo host/bin/setup-vm delete #{q_vm}") unless vm.load_balancer
```

Comment:
I think this could be an issue:
1. we delete a vm with a load balancer, i.e. we don't execute `setup-vm delete`
2. in the next step we return storage, cpu, memory to the host but the vm still occupies them as we haven't executed `setup-vm delete`
3. in `wait_vm_removal_from_load_balancer` we finally call `setup-vm delete` to actually free all resources the vm occupied on the host

between 2 and 3, the allocator could overprovision the host

Reply:
Switched back to the old implementation